src/Components/Bookpage/index.js

```diff
@@ -32,6 +32,11 @@ function Bookpage() {
                     setClaimed(bookData.data.claimed_by_name)
                 }
             })
+
+    }
+
+    useEffect(() => {
+        getBookData()
     }, [id, error])
```

Comment:
Now you can actually scrap the fat arrow and just do

`useEffect(getBookData,[....])`

Reply:
done